### DISCUSSION TITLE
[FR] support mkdocs-material, docusaurus

### DISCUSSION THREAD

**Discussion by andrey-jef** [songkg7/o2#346] [Ideas]
**Is your feature request related to a problem? Please describe.**

No. Just want to express my interest to the conversion to [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) and [docusaurus](https://docusaurus.io/)

**Describe the solution you'd like**
N/A

**Describe alternatives you've considered**

- mkdocs-material: https://github.com/ObsidianPublisher/obsidian-github-publisher
- docusaurus: N/A

**Additional context**

**Comment by songkg7** [comment]
Thanks for your suggestions! I will review the contents and share the progress.

However, this plugin has not yet been released to the community, so it may be pushed back.

please refer to the [PR](https://github.com/obsidianmd/obsidian-releases/pull/1678) for obsidian community-plugin release.

**Comment by songkg7** [comment]
since 2.0, support docusaurus.
